src/renderer/wg_engine/tvgWgRenderTarget.cpp

```diff
@@ -173,12 +173,34 @@ void WgRenderStorage::clear(WGPUCommandEncoder commandEncoder)
 }
 
 
-void WgRenderStorage::blend(WGPUCommandEncoder commandEncoder, WgRenderStorage* targetSrc, WgBindGroupBlendMethod* blendMethod)
+void WgRenderStorage::blendSolid(WGPUCommandEncoder commandEncoder, WgRenderStorage* targetSrc, WgBindGroupBlendMethod* blendMethod, WgBindGroupOpacity* opacity)
```

Comment:
can you unify these 3 (pass WgPipelineBlend as a function arg/template) so we don;t have code that is almost identical?

Reply:
Yes, make a sence. Fixed